SparkGen/index.js

```diff
@@ -109,7 +113,12 @@ $(document).ready(function () {
 
 //handling reading dataframe source
 function readDataFrame(sourceType, load, filePath) {
-    return 'df = spark.read.format("' + sourceType + '").' + load + '("' + filePath + '")';
+    // return 'df = spark.read.format("' + sourceType + '").' + load + '("' + filePath + '")';
+    return 'df = spark.read.format("' + sourceType + '").' + load + '("' + filePath + '.'+ load +'")';
+}
+
+function readConditions(selectedCols){

```

Comment:
rename function to readSort. There is a separate dropdown for conditions for which we can add a new function.